spec/api/houdini/nonprofit_spec.rb

```diff
@@ -113,7 +113,7 @@ def h(h = {})
           user: {name: "Name", email: "em@em.com", password: "12345678", password_confirmation: "12345678"}
       }
 
-      xhr :post, '/api/v1/nonprofit', input
+      post '/api/v1/nonprofit', params: input, xhr: true
```

Comment:
xhr and the use of positional params in request specs is deprecated so this corrected that.